DependencyInjection/DoctrineExtension.php

```diff
@@ -477,6 +492,7 @@ protected function ormLoad(array $config, ContainerBuilder $container)
     protected function loadOrmEntityManager(array $entityManager, ContainerBuilder $container)
     {
         $ormConfigDef = $container->setDefinition(sprintf('doctrine.orm.%s_configuration', $entityManager['name']), new ChildDefinition('doctrine.orm.configuration'));
+        $ormConfigDef->addTag(IdGeneratorPass::CONFIGURATION_TAG);
```

Comment:
Tags are not inherited so adding it to the XML won't help here. Also yes, this is mostly internal, flagging some services so that IdGeneratorPass can collect them easily.